crates/rust-analyzer/src/config.rs

```diff
@@ -901,6 +901,16 @@ impl Config {
                 serde_json::Error::custom("expected a non-empty string"),
             ));
         }
+        if self.has_linked_projects() {
+            for path in self.linked_manifests() {
+                if let Err(e) = ProjectManifest::from_manifest_file(self.root_path.join(path)) {
```

Comment:
This doesn't check the pointed to file for validity, only whether the path is absolute (which it always will be given the join)